.gitignore

```diff
@@ -27,7 +27,12 @@ tools/Cake
 packages/
 # !packages/repositories.config
 
+# NUnit results
+TestResult.xml
 
+# Tools installed by Cake
+tools/
+!tools/packages.config
```

Comment:
That only happens if it's not present. In the past, we have edited this file to get a new version locally. The download of packages.config is a new feature in the bootstrap - at least new since 0.6.4 - so I'm not sure what it downloads. However, it seems to me that we have to control the version of our tools that get used.